src/ledger/LedgerStateOfferSQL.cpp

```diff
@@ -278,6 +277,56 @@ LedgerStateRoot::Impl::loadOffers(StatementContext& prep) const
     return offers;
 }
 
+std::list<LedgerEntry>::const_iterator
+LedgerStateRoot::Impl::loadOffers(StatementContext& prep,
+                                  std::list<LedgerEntry>& offers) const
+{
+    auto iterNext = --offers.cend();
```

Comment:
I think this is UB if `offers` is empty

Reply:
You are indeed correct, simplified the implementation to avoid this.